continuous_integration/runtests.sh

```diff
@@ -2,5 +2,7 @@
 set -xe
 
 cd hdfscm
+export CLASSPATH=`$HADOOP_HOME/bin/hadoop classpath --glob`
+export JUPYTER_ENV=dev
```

Comment:
```suggestion
export JUPYTER_ENV=test
```

It's a bit more clear.